Tests/TABResourceLoaderTests/Helpers/XCTestCase+Additions.swift

```diff
@@ -33,11 +33,24 @@ extension XCTestCase {
       }
     }
   }
-  
+
   /// Convenience function for waiting for an expectation, timeout defaults to 1
   func waitForExpectation(file: StaticString = #file, line: UInt = #line) {
     waitForExpectations(timeout: 1) { error in
       if let error = error { XCTFail("\(error)", file: file, line: line) }
     }
   }
+
+  /// Convenience function to serialize a JSON object, if serilization fails an XCTest assertion is raised
+  ///
+  /// - Parameter obj: The object to serialize
```

Comment:
I renamed `obj` to `jsonObject`, the `file` and `line` parameter are intentionally not documented since they not meant to be used, the purpose is to move the XCTest failure to the function that called it. 

For example, the interface for `XCTFail` is `public func XCTFail(_ message: String = default, file: StaticString = #file, line: UInt = #line)` but the docs only mention the `message` parameter.